env_aop.yaml

```diff
@@ -15,6 +15,7 @@ parameters:
   ssh_user: cloud-user
   master_docker_volume_size_gb: 25
   node_docker_volume_size_gb: 25
+  docker_veresion: 1.8.2
```

Comment:
I would prefer to leave docker_version to the empty string by default to avoid having to follow the required docker version in our templates